docker/update-all.sh

```diff
@@ -1,16 +1,17 @@
 #!/bin/bash
 
-./oval.sh --redhat ${@} 
-./oval.sh --amazon ${@} 
-./oval.sh --debian ${@} 
-./oval.sh --ubuntu ${@} 
-./oval.sh --alpine ${@} 
-./oval.sh --oracle ${@} 
-./gost.sh --redhat --batch-size 500 ${@} 
-./gost.sh --debian --batch-size 500 ${@} 
-./gost.sh --ubuntu --batch-size 15 ${@} 
-./nvd.sh ${@} 
-./jvn.sh ${@} 
-./exploitdb.sh ${@} 
-./msfdb.sh ${@} 
-./kev.sh ${@} 
+./oval.sh --redhat ${@}
+./oval.sh --amazon ${@}
+./oval.sh --debian ${@}
+./oval.sh --ubuntu ${@}
+./oval.sh --alpine ${@}
+./oval.sh --oracle ${@}
+./oval.sh --fedora ${@}
+./gost.sh --redhat --batch-size 500 ${@}
+./gost.sh --debian --batch-size 500 ${@}
+./gost.sh --ubuntu --batch-size 15 ${@}
+./nvd.sh ${@}
+./jvn.sh ${@}
+./exploitdb.sh ${@}
+./msfdb.sh ${@}
+./kev.sh ${@}
```

Comment:
![image](https://user-images.githubusercontent.com/49754654/151473097-2010db4a-b304-4608-8ee7-f2d0fcbeebfb.png)

Certainly Line feed code is LF, why...

Reply:
This command seems to be related to space.
```console
$ git diff support_fedora:docker/update-all.sh..master:docker/update-all.sh
```